src/modules/fancyzones/lib/WindowMoveHandler.cpp

```diff
@@ -316,7 +318,14 @@ void WindowMoveHandlerPrivate::MoveSizeEnd(HWND window, POINT const& ptScreen, c
     {
         auto zoneWindow = std::move(m_zoneWindowMoveSize);
         ResetWindowTransparency();
-        zoneWindow->MoveSizeEnd(m_windowMoveSize, ptScreen);
+        auto windowInfo = FancyZonesUtils::GetFancyZonesWindowInfo(window);
+        auto myTie = [](const FancyZonesUtils::FancyZonesWindowInfo& windowInfo) {
+            return std::tie(windowInfo.noVisibleOwner, windowInfo.processPath, windowInfo.standardWindow);
+        };
+        if (myTie(windowInfo) == myTie(m_moveSizeStartWindowInfo))

```

Comment:
This might be fine, but I would feel more comfortable if we do an explicit check for the specific case we know of and only bail out only if `windowInfo.noVisibleOwner` and `windowInfo.standardWindow` change from true to false.
Also, let's not check `windowInfo.processPath` because as far as we know, this is not a factor.
Also add a comment explaining what case we are covering with that check.